docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1953,9 +1953,20 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/core/extendedCodeableConceptValidation`|boolean|A boolean flag which indicates whether extended validation is performed by the server during object construction for code, Coding, CodeableConcept, Quantity, Uri, and String elements which have required bindings to value sets.|
 |`fhirServer/core/disabledOperations`|string|A comma-separated list of operations which are not allowed to run on the IBM FHIR Server, for example, `validate,import`. Note, do not include the dollar sign `$`|
 |`fhirServer/core/defaultPageSize`|integer|Sets the pageSize to use in search and history when no _count parameter is specified in the request. If a user-specified value exceeds the max page size (1000), then a warning is logged and max page size will be used. If not provided, the default page size (10) is used.|
+|`fhirServer/term/disableCaching`|boolean|Indicates whether caching is disabled for the FHIR terminology module, this includes caching in `CodeSystemSupport` and `ValueSetSupport`|
 |`fhirServer/term/graphTermServiceProvider/enabled`|boolean|Indicates whether the graph term service provider should be used by the FHIR term service to access code system content|
 |`fhirServer/term/graphTermServiceProvider/timeLimit`|integer|Graph traversal time limit (in milliseconds)|
 |`fhirServer/term/graphTermServiceProvider/configuration`|object (name/value pairs)|A JSON object that contains the name/value pairs used to configure the graph database behind the graph term service provider see: [https://docs.janusgraph.org/basics/configuration-reference/](https://docs.janusgraph.org/basics/configuration-reference/)|
+|`fhirServer/term/remoteTermServiceProvider/enabled`|boolean|Indicates whether this remote term service provider should be used by the FHIR term service to access code system content|
+|`fhirServer/term/remoteTermServiceProvider/base`|string|The base URL for this remote term service provider|
+|`fhirServer/term/remoteTermServiceProvider/trustStore/location`|string|The trust store location for this remote term service provider|
+|`fhirServer/term/remoteTermServiceProvider/trustStore/password`|string|The trust store password for this remote term service provider|
+|`fhirServer/term/remoteTermServiceProvider/trustStore/type`|string|The trust store type (e.g. pkcs12) for this remote term service provider|
+|`fhirServer/term/remoteTermServiceProvider/basicAuth/username`|string|The basic authentication username for this remote term service provider|
+|`fhirServer/term/remoteTermServiceProvider/basicAuth/password`|string|The basic authentication password for this remote term service provider|
+|`fhirServer/term/remoteTermServiceProvider/httpTimeout`|integer|The HTTP read timeout for this remote term service provider (in milliseconds)|
+|`fhirServer/term/remoteTermServiceProvider/supports/system`|string|The system URI supported by this remote term service provider, NOTE: there can be more than one `supports` element|
```

Comment:
so is supports an array then?  I think we've avoided arrays of objects in our config up til now.  wondering if we should update this to be more clear.

Reply:
an alternative would be for this to be a list of canonical urls.  for example:
```
"supports": [
    "http://snomed.info/sct", 
    "http://loinc.org|2.69"
]
```

Reply:
You are correct. The `supports` element is an array of objects. I know that this challenges the status quo a bit. I never understood why we have tried to actively avoid arrays of Json objects in our configuration file. I have updated the docs to clarify.

Reply:
> I never understood why we have tried to actively avoid arrays of Json objects in our configuration file.

Just a couple reasons I think:
1. Not great support from our FHIRConfigHelper
2. It ties our implementation to JSON config...making it harder to get these config values from other config providers in the future